app/src/main/java/agency/five/codebase/android/movieapp/ui/component/ActorCard.kt

```diff
@@ -0,0 +1,78 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.ui.theme.*
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+import coil.request.ImageRequest
+
+data class ActorCardViewState(
+    val imageUrl: String,
+    val name: String,
+    val character: String,
+)
+
+@Composable
+fun ActorCard(
+    actorCardViewState: ActorCardViewState,
+    modifier: Modifier = Modifier,
+) {
+    Card(
+        shape = RoundedCornerShape(10),
+        elevation = 10.dp,
+        modifier = modifier.width(130.dp)
+    ) {
+        Column{
+            AsyncImage(
+                model = ImageRequest.Builder(LocalContext.current)
+                    .data(actorCardViewState.imageUrl)
+                    .build(),
+                contentDescription = "${actorCardViewState.name} profile image",
+                modifier = modifier
+                    .fillMaxWidth()
+                    .height(130.dp),
+                contentScale = ContentScale.Crop
+            )
+            Text(
+                text = actorCardViewState.name,
+                fontWeight = FontWeight.ExtraBold,
+                modifier = modifier
+                    .padding(start = 10.dp, top = 6.dp, end = 30.dp, bottom = 2.dp),
+                fontSize = 12.sp
+            )
+            Text(
+                text = actorCardViewState.character,
+                color = if (isSystemInDarkTheme()) Gray300 else Gray600,
```

Comment:
If you want to have theming in your app, this is done in `Theme.kt` file. Colors should be defined in `DarkColorPalette` and `LightColorPalette`, and are accessed with `MaterialTheme.colors`. That way there is only one if statement which decides if system uses dark or light theme, and all colors need to be changed only in one place.